meteor-app/imports/api/members/members.js

```diff
@@ -20,18 +53,84 @@ Members.attachSchema(new SimpleSchema({
     label: "Is signed in",
     defaultValue: false,
   },
+  joined: {
+    type: Date,
+    label: "Date added to database",
+    defaultValue: new Date(),
+  },
   lastIn: {
     type: Date,
-    label: "Date of last session",
-    optional: true
+    label: "Date of last interaction",
+    defaultValue: new Date(),
   },
   sessions: {
     type: [Object],
     label: "Array of sessions attended",
-    optional: false,
     defaultValue: [],
     blackbox: true,
   },
+  sessionCount: {
+    type: Number,
+    defaultValue: 0,
+  },
+  bikesHousehold: {
+    type: Number,
+    label: "Number of bikes in household",
+    optional: true,
+  },
+  primaryBike: {
+    type: String,
+    // allowedValues: [
```

Comment:
Relaxed rules, to allow for the import of things not in this list - may need to migrate that data before turning the rule back on